guides/getting-started/using-dev-tunnels.md

```diff
@@ -0,0 +1,109 @@
+# Using dev tunnels <Badge type="warning" text="client" /><Badge type="danger" text="server" />
+
+When you are developing an Asp.Net or Asp.Net Core based web application running on your developer machine, usually you can connect to the server using the pre configured `applicationUrl` (e.g https://localhost:44359)
+
+If you are, however, integrating an Android or iOS application using MAUI or Avalonia, the app will run either on a physical device or on a local emulator/simulator. In either case using localhost is not an option.
+
+Instead you can use [Microsoft dev tunnels](https://learn.microsoft.com/en-us/azure/developer/dev-tunnels/overview).
+
+Both tools provide you with a URL that you can connect to. Any traffic received will be routed to your local web application.
+In this guide we will guide you in using those tools.
+
+::: info
```

Comment:
If it's not a recommended thing, let's just remove that to avoid tempting users to do the incorrect thing 😄

Reply:
you mean the info block?

Reply:
Yeah, the whole block (that says using the `10.0.2.2` address can be used but has side effects). Unless I misunderstood what the block really means?